spec/models/shared_examples/shared_examples_for_archivable.rb

```diff
@@ -82,35 +87,27 @@ def check_log_for_error(error_message)
 
   def archive_test(initial_count,
                    count_total,
-                   _count_greater_equal_production,
                    archive_count)
     expect(original_type.count).to eq(initial_count)
     expect(archived_type.count).to eq(0)
 
     Archiver.archive_previous_versions
 
+
     expect(original_type.count).to eq(count_total)
     expect(archived_type.count).to eq(archive_count)
-  end
-
-  def create_production_version
-    create :version, :preview
-    Version.current_preview.update(production: true)
-    create :institution, version_id: current_production_id
-    create factory, institution_id: Institution.last.id
-  end
-
-  def create_preview_version
-    create :version, :preview
-    create :institution, version_id: current_preview_id
-    create factory, institution_id: Institution.last.id
-  end
 
-  def current_preview_id
-    Version.current_preview.id
+    expect(original_type.distinct.pluck(:version_id)).to include(*[Version.current_production.id, Version.current_preview&.id].compact)
+    expect(archived_type.pluck(:id)).not_to include(*original_type.pluck(:id))
```

Comment:
Check that `archived` and `current` records were separated. (i.e. no `current` records in `archived`).
I consider this test "OK, but not ideal".  The records with `institutions` associations can't be tied back to `version` right now due to missing associations among `archives` tables.